pkg/ovs/openflow/ofctrl_packetin.go

```diff
@@ -20,6 +20,14 @@ import (
 	"github.com/contiv/ofnet/ofctrl"
 )
 
+const (
+	NPPacketInMeterId = 1
+	TFPacketInMeterID = 2
+
+	NPPacketInRate = 100
+	TFPacketInRate = 200
```

Comment:
I think we use 100 for the TF packet-in queue in the Antrea Agent, any reason why we use a different value here

Reply:
I was thinking that we don't want TraceFlow traffic to be dropped. So I put a larger and safer number for TraceFlow. I don't have a strong opinion on this number. I can change it to 100.

Reply:
200 is fine if we want to accommodate for a small burst, but I don't see a compelling reason to have a different rate for the NP and TF rates

Reply:
Done.